src/tools/compiletest/src/runtest.rs

```diff
@@ -1280,6 +1280,12 @@ actual:\n\
         let extra_link_args = vec!["-L".to_owned(),
                                    aux_dir.to_str().unwrap().to_owned()];
 
+        let mut env = self.props.rustc_env.clone();
+        // Tell emscripten to link using libc produced with LLVM backend
+        if self.config.target.contains("wasm32") && self.config.target.contains("experimental") {
+            env.push(("EMCC_WASM_BACKEND".to_string(), "1".to_string()));
```

Comment:
Oh yeah sure having this is fine, it just seems like it should be in rustc's backend (e.g. the invocation of the linker) rather than only in compiletest.